### PR TITLE
feat: DTOSS-4840 move rules files out of blob

### PR DESCRIPTION
**Pull request by sudipnhs**
## Description
--Moved the rules files out of blob storage

## Context

[DTOSS-4840](https://nhsd-jira.digital.nhs.uk/browse/DTOSS-4840)

## Type of changes

- [ ] Refactoring (non-breaking change)
- [ ] New feature (non-breaking change which adds functionality)
- [x] Breaking change (fix or feature that would change existing functionality)
- [ ] Bug fix (non-breaking change which fixes an issue)

## Checklist

- [ ] I am familiar with the [contributing guidelines](../docs/CONTRIBUTING.md)
- [x] I have followed the code style of the project
- [x] I have added tests to cover my changes
- [ ] I have updated the documentation accordingly
- [ ] This PR is a result of pair or mob programming

---

## Sensitive Information Declaration

To ensure the utmost confidentiality and protect your and others privacy, we kindly ask you to NOT including [PII (Personal Identifiable Information) / PID (Personal Identifiable Data)](https://digital.nhs.uk/data-and-information/keeping-data-safe-and-benefitting-the-public) or any other sensitive data in this PR (Pull Request) and the codebase changes. We will remove any PR that do contain any sensitive information. We really appreciate your cooperation in this matter.

- [x] I confirm that neither PII/PID nor sensitive data are included in this PR and the codebase changes.

### CODE REVIEW

File: application/CohortManager/src/Functions/Shared/Common/Common.sln
Comment (MWClayson-NHS) [inline application/CohortManager/src/Functions/Shared/Common/Common.sln:1]: Please remove this file. there should only be one solution file in the project

File: application/CohortManager/src/Functions/Shared/dotnet-mesh-client
Comment (MWClayson-NHS) [inline application/CohortManager/src/Functions/Shared/dotnet-mesh-client:1]: Can you run `Git Submodule --update` ? that should resolve this differece.

File: application/CohortManager/src/Functions/Shared/Common/ReadRules.cs [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+            }
+
+            // Read the JSON file content as a string
+            string jsonContent = await File.ReadAllTextAsync(filePath);
+            return jsonContent;
+        }
+        catch (Exception ex)
+        {
+            _logger.LogError(ex, "error while getting rules from directory: {ex} {fileName}", ex.Message, jsonFileName);
+            throw;
```
Comment (andrewmorris43) [inline application/CohortManager/src/Functions/Shared/Common/ReadRules.cs:41]: Should this raise a system exception here instead of just throw?

Comment (sudipnhs) [reply application/CohortManager/src/Functions/Shared/Common/ReadRules.cs]: There shouldn't be system exception. It should log the error if it encounters error. I used "throw" while testing it locally but forgot to remove "throw". No, it's remove and good to go.

**Review by andrewmorris43 — APPROVED** [review]

**Review by MWClayson-NHS — APPROVED** [review]

**Pull request merged**